shard/src/api/api.py

```diff
@@ -73,8 +77,36 @@ async def set_response(self, user_id, guild_id, trigger, response):
 
     @fetch_guild
     async def get_playlist(self, guild):
-        voice = self.bot.cogs['Voice'].voice_managers[guild.id]
-        return voice.q.as_dict(), sc.OK_200
+        voice = self.bot.lavalink.player_manager.get(guild)
+        if voice is None:
+            return {}, sc.OK_200
+        else:
+            songs = map(LavaSong, voice.queue)
+            dicts = list(map(methodcaller('as_dict'), songs))
+            return {'songs': dicts}, sc.OK_200
+
+    @fetch_guild
+    async def queue_song(self, guild_id, uid, song):
+        lava_cog = self.bot.cogs['Voice']
+        guild = self.bot.get_guild(guild_id)
+        if guild is None:
```

Comment:
if you're using `@fetch_guild` this is done automatically

Reply:
also your first parameter `guild_id` contains a `Guild` object already, as written